public/assets/js/allcount.js

```diff
@@ -154,7 +158,10 @@ allcountModule.config(["fieldRenderingServiceProvider", function (fieldRendering
                     locale: $locale.id.split("-")[0],
                     format: toMomentDateFormat(dateFormat(fieldDescription))
                 });
-                input.data("DateTimePicker").date(parseDate(controller.$viewValue) || null);
+                if(typeof controller.$viewValue != 'undefined') {
+                    input.data("DateTimePicker").date(parseDate(controller.$viewValue) || null);
+                    input.data("DateTimePicker").defaultDate(parseDateToMoment(controller.$viewValue) || null);
```

Comment:
@stuhol Could you please explain the use case of setting up `defaultDate` manually?